index.js

```diff
@@ -145,46 +198,63 @@ function nowServing(line) {
 
 const line = []
 
+
+
 // ***** Scope & Closures - Question 1 *****
+let ticket = 0
 
+const takeATicketNumber = (line) => {
+	ticket ++
+	line.push(ticket)
+	return `"Welcome. You are ticket number ${ticket}"`
+}
 
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+console.log("%cScope & Closures - Question 1", "color: red")
 
-// console.log(takeATicketNumber(line))
+console.log(takeATicketNumber(line))
 // // => `Welcome. You are ticket number 1`
 
-// console.log(takeATicketNumber(line))
+console.log(takeATicketNumber(line))
 // // => `Welcome. You are ticket number 2`
 
-// console.log(nowServing(line))
+console.log(nowServing(line))
 // // => `Currently serving 1.`
 
-// console.log(nowServing(line))
+console.log(nowServing(line))
 // // => `Currently serving 2.`
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
-// console.log(takeATicketNumber(line))
+console.log(takeATicketNumber(line))
 // // => `Welcome. You are ticket number 3`
 
 
 // ***** Scope & Closures - Question 2 *****
+function ticketNumberGeneratorFunc(ticket = 0) {
+	function takeATicketNumber(line) {
+		ticket ++
+		line.push(ticket)
+		return `"Welcome. You are ticket number ${ticket}"`
+	}
+	return takeATicketNumber
+}
```

Comment:
This is awesome, great work getting the closure working! I like your solution with the default argument as well. That would mean you could use this function to start a line at any line number you wanted. 💯